bloges56/DateSim
# Bl arcadev2

## Description
New version of arcade that includes NPCs with animations and player animation.  Also includes basic arcade cabinets.

## How To Test
- Make sure that all the game scenes are in your build settings
- Run the SceneManager scene and click start game
- Interact with all the NPCs and objects in the scene, rerunning the scene as need be

## Checklist
- [x] Do both NPCs load the dialogue scene?
- [x] Does the circle object act as before?
- [x] Do the "arcade cabinets" load their respective scenes?
